### PR TITLE
Buenos aires source

### PR DESCRIPTION
**Pull request by mfogel**
This is a new source for Buenos Aires neighborhood, borough and city limits data - see whosonfirst-data/whosonfirst-data#180

Of note:

 * I added support for UTF-8 characters that aren't the basic characters to the 'make' process. This is required by the accented characters in the name `Ciudad Autónoma de Buenos Aires` and those in the fragment of license text
 * the short name of the source `ar-caba` comes from: AR for 'argentina' obviously, and 'CABA' is the most common abbreviation of the `Ciudad Autónoma de Buenos Aires`. This is what I've seen this used in mailing addresses. I would recommend against using just `ba` because this is ambiguous with the Province of Buenos Aires.

Not sure if you'd prefer the license fragment be translated to english or not. As of now, I left it in spanish.

### CODE REVIEW

**Comment by nvkelso** [comment]
@mfogel Can you use a 3 character country code prefix, so: `arg-caba`, please? We're just starting to standardize on that convention – when we had less sources it was less of an issue ;)

File: sources/ar-caba.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+	"id": 1108969549,
+	"fullname": "Ciudad Autónoma de Buenos Aires, Iniciativa de Datos Públicos y Transparencia",
+	"name": "ar-caba",
+	"prefix": "ar-caba",
```
Comment (nvkelso) [inline sources/ar-caba.json]: @stepps00 would this be `arg-caba` or just `caba`?

Comment (stepps00) [inline sources/ar-caba.json]: this would be `arg-caba`, similar to how we prefix `can` for Canadian sources.

**Comment by stepps00** [comment]
@mfogel - this is great! I only have two comments:

- We should change `ar-caba` to `arg-caba` and rerun the makefile. The makefile _should_ update the associated .json and .md files once the source prefix is changed, but if not, I can update the repo/source with information you provided.

- Thanks for the utf-8 support in the `docs.py script!

**Comment by stepps00** [comment]
This PR looks good, ready to merge.

**Pull request merged**